src/lib/OpenEXRCore/debug.c

```diff
@@ -87,7 +87,7 @@ print_attr (const exr_attribute_t* a, int verbose)
                 "dwab",
                 "htj2k"};
             printf (
-                "'%s'", (a->uc < 10 ? compressionnames[a->uc] : "<UNKNOWN>"));
+                "'%s'", (a->uc < 11 ? compressionnames[a->uc] : "<UNKNOWN>"));
```

Comment:
```suggestion
                "'%s'", (a->uc < EXR_COMPRESSION_LAST_TYPE ? compressionnames[a->uc] : "<UNKNOWN>"));
```

Wouldn't the 11 be better as `EXR_COMPRESSION_LAST_TYPE`?

Reply:
Thanks for mentioning this enum constant! Suggestion applied.

I tested a local build and it works well.